tests/test_smooth_field.py

```diff
@@ -0,0 +1,72 @@
+# Copyright 2020 - 2021 MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+
+import numpy as np
+import torch
+from parameterized import parameterized
+
+from monai.transforms import RandSmoothFieldAdjustContrastd, RandSmoothFieldAdjustIntensityd
+from tests.utils import assert_allclose, is_tf32_env
+
+_rtol = 5e-3 if is_tf32_env() else 1e-4
+
+INPUT_SHAPE1 = (1, 8, 8)
+INPUT_SHAPE2 = (2, 8, 8)
+
+TESTS_CONTRAST = [
```

Comment:
Could you make sure that you test np, torch and torch.cuda input (eg https://github.com/Project-MONAI/MONAI/blob/dev/tests/test_crop_foreground.py#L23)?